service.py

```
@@ -96,3 +98,14 @@ async def random_possess(
 
         await self.poke_user(client, group_id, target_id)
         return target_id, target_name
+
+    async def set_avatar(self, client, target_id):
+        payloads = {
+            "file": f"https://q1.qlogo.cn/g?b=qq&nk={target_id}&s=640"
+        }
+        try:
+            await client.api.call_action('set_qq_avatar', **payloads)
+            return True
```

Ruff (0.14.14)

[warning] Consider moving this statement to an `else` block

(TRY300)